pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -42,6 +41,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	DataMountPath        = "/opt/infinispan/server/data"
+	IdentitiesVolumeName = "identities-volume"
+	ConfigVolumeName     = "config-volume"
+
+	ConditionPrelimChecksFailed = "preliminaryChecksFailed"
+	ConditionGracefulShutdown   = "gracefulShutdown"
+	ConditionStopping           = "stopping"
```

Comment:
This should be enum in infinispan_types.go and plant to be implemented in #482

Reply:
Makes sense. I have move the string constants to `infinispan_types.go`. I haven't converted them to enum consts, as that requires greater changes which should be solved by a #482 PR.